docs/en/Command-Line-Interface.md

```diff
@@ -243,6 +243,16 @@ For example, `ls` can be used to browse the file system.
 
 `ls` loads the metadata for immedidate children of a directory.
 
+### masterInfo
+
+The `masterInfo` command prints the relative information of the Alluxio masters. If Alluxio is running in single
```

Comment:
hm, it looks like the shell also documents in the same way. It might be clearer to instead say "The `masterInfo` command prints information regarding master fault tolerance such as leader address, list of master addresses, and the configured Zookeeper address"